samtranslator/model/s3_utils/uri_parser.py

```diff
@@ -85,17 +85,7 @@ def construct_s3_location_object(
 
         s3_pointer = location_uri
 
-    else:
```

Comment:
Why was this removed?

Reply:
oh I think this was when I resolved merge conflicts here. One sec